src/front/app/components/Create/Create.js

```diff
@@ -0,0 +1,151 @@
+                            <span className="challenge-create-basic__address">
+                                <img className="challenge-create-basic__img-address" src="" alt="img" />75 Zhylyanska St, Kyiv, Ukraine
+                            </span>
+                            <div className="challenge-create-disabled__title">Online
+                                <div className="challenge-create-disabled__checkbox"><input type="checkbox" checked  disabled/></div>
+                            </div>
+                            
+                                <div className="challenge-create-basic__description-title">Description</div>
+                            <hr/>
+                           <textarea placeholder="Tell Users about the background of the issue." className="challenge-create-basic__description"></textarea>
+                          {description.touched && description.error && <div className="form-error">{description.error}</div>}
+                        </div>
+
+                    <div className="challenge-create-block">
+                        <CreateTitleBlock text="Settings" />
+
+                        <div className="challenge-create-select-title">
+                            Category
+
+                            <select className="challenge-create-select-block">
+                                <option value="-1"> > </option>
+                                <option>Category 1</option>
+                                <option>Category 2</option>
+                                <option>Category 3</option>
+                            </select>
+                        </div>
+                        <EnabledCheckbox text="Repeateble" />
+                        <DisabledCheckbox text="End date" />
+                        <EnabledCheckbox text="Restrict Visibility" />
+
+                        <SetDay text="Start Day"></SetDay>
+                        <SetDay text="End Day"></SetDay>
+                    </div>
+                        <div className="challenge-create-block">
+                        <CreateTitleBlock text="Requirements" />
+
+                        <EnabledCheckbox text="Financial Recources" />
+                        <DisabledCheckbox text="Special Skills" />
+                        <DisabledCheckbox text="Spare Time" />
+
+                        <div className="challenge-create-select-title">
+                            Complexity level
+
+                            <select className="challenge-create-select-block">
+                                <option value="-1"> > </option>
+                                <option>Low</option>
+                                <option>Middle</option>
+                                <option>Hight</option>
+                            </select>
+
+
+                        <EnabledCheckbox text="Verification" />
+                         </div>
+                        </div>
+                        <div className="challenge-create-block">
```

Comment:
remove -block wording from class names, it is already seen that it is block,
for example we did not write 
 ```js 
const textConstant = 'some text';  // bad
const text = 'some text'; // good
```
same for css
```scss
// bad
.user-block {
    &__name-element {
    }
}
// good
.user {
  &__name {
  }
}
```